packages/pebble-web/src/components/Slider.tsx

```diff
@@ -31,7 +33,7 @@ const Slider: React.FunctionComponent<SliderProps> = ({
   const _values = Array.isArray(values) ? values.slice(0) : values || [];
 
   if (Array.isArray(values)) {
-    if (!values[0] && rest.min) {
+    if (!values[0] && rest.min !== undefined) {
       _values[0] = rest.min;
     }
```

Comment:
new if condition will pass when `rest.min === 0`. is that ok?

Reply:
yes.